README.md

```diff
@@ -60,36 +60,25 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 Hello Microverse! <a name="about-project"></a>
 
 > Describe your project in 1 or 2 sentences.
```

Comment:
You did well implementing the project requirement and adding the README file usimg the required template, kindly remove all descriptions from your README file as they were added to give you instructions on what to do and not as part of your file. The descriptions in your README usually starts with `>`, please kindly remove all of them 😄